src/test/java/org/selenium/pages/tests/BaseClass.java

```diff
@@ -0,0 +1,53 @@
+package org.selenium.pages.tests;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.selenim.pages.LogInPage;
+import org.selenim.pages.ProfilePage;
+
+import java.time.Duration;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+public abstract class BaseClass {
+    protected static WebDriver driver;
+    protected static LogInPage logInPage;
+    protected static ProfilePage profilePage;
```

Comment:
This class shouldn't know anything about specific page. It should have common code that applicable to all tests. Also it can have methods that can be reused by tests